src/hooks/profile/use-change-password.hook.ts

```
@@ -0,0 +1,21 @@
+
+// Custom hook to handle the change password process
+export function useChangePassword() {
+  return useMutation({
+    // Function that performs the password change request
+    mutationFn: changePassword,
+    // Triggered when the password is successfully changed
+    onSuccess: () => {
+      toast.success("Password updated successfully!");
+    },
+    // Triggered when the API call fails
+    onError: (error: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)